reactor-netty-http/src/main/java/reactor/netty/http/server/MicrometerHttpServerMetricsRecorder.java

```diff
@@ -162,4 +187,20 @@ public void recordConnectTime(SocketAddress remoteAddress, Duration time, String
 	public void recordResolveAddressTime(SocketAddress remoteAddress, Duration time, String status) {
 		throw new UnsupportedOperationException();
 	}
+
+	private LongAdder getServerConnectionAdder(String uri, String method) {
```

Comment:
transitively, this should also only have a single input parameter `SocketAddress localAddress`.